src/lgtm/scripts/download/grafana.sh

```
@@ -0,0 +1,52 @@
+esac
+
+# Prepare
+mkdir -p /tmp/grafana
+cd /tmp/grafana
+
+# Download
+download "$REPO/${ARCHIVE}"
+download "$REPO/${ARCHIVE}.sha256"
+
+# Verify
+echo "$(cat ${ARCHIVE}.sha256)  ${ARCHIVE}" | sha256sum --check --status
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)